Base/res/keymaps/da.json

```diff
@@ -0,0 +1,6 @@
+{
+  "map":       [ "", "\u001B", "1",  "2", "3", "4", "5", "6", "7", "8", "9", "0", "+", "´", "\b",   "", "q", "w", "e", "r", "t", "y", "u", "i", "o", "p", "å", "¨", "\n", "", "a", "s", "d", "f", "g", "h", "j", "k", "l", "æ", "ø", "½", "", "'", "z", "x", "c", "v", "b", "n", "m", ",", ".", "-", "", "*", "", " ", "", "", "", "", "", "", "", "", "", "", "", "", "", "7", "8", "9", "-", "4", "5", "6", "+", "1", "2", "3", "0", ",", "", "",  "<", "", "", ""],
```

Comment:
Shouldn't it be "\t" before "q" instead of ""?

Reply:
Yes, indeed it should! I seem to have swapped _map_ and _shift_map_ here; I'll get on it right away (or perhaps after I've eaten dinner)

Reply:
That should do it!

Reply:
Looks good to me! BTW very cool that you took the time to make this! :^)

Reply:
Thanks! :^D